### PR TITLE
Pass login specs and cleanup

### PR DESCRIPTION
**Pull request by bshyong**
- Adds MFA to user profile, resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4819
- Adds fixes for some specs

### CODE REVIEW

**Comment by bshyong** [comment]
cc @U-DON for visibility re: specs

**Comment by bshyong** [comment]
@U-DON this is ready to look at re: MFA additions

File: src/js/user-profile/components/UserDataSection.jsx [hunk 1]
```diff
@@ -69,13 +87,36 @@ class UserDataSection extends React.Component {
     );
   }
 
+  renderMultifactorMessage() {
```
Comment (U-DON) [inline src/js/user-profile/components/UserDataSection.jsx]: Should this check for `this.props.profile.multifactor`?

**Review by U-DON — APPROVED** [review]
LGTM.

File: src/js/user-profile/components/UserDataSection.jsx (hunk 1 above)
Comment (bshyong) [reply src/js/user-profile/components/UserDataSection.jsx]: Fixed. Also added some fixes for the E2E tests

**Comment by bshyong** [comment]
Specs are passing again including E2E tests

File: src/js/signin/components/Verify.jsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ class Verify extends React.Component {
           <div className="row">
             <div className="columns small-12">
               <p>
-                We'll need to verify your identity so that you can securely access and manage your benefits.<br/>
+                We’ll need to verify your identity so that you can securely access and manage your benefits.<br/>
```
Comment (U-DON) [inline src/js/signin/components/Verify.jsx]: Why were these replaced with the curly single quotes? Should probably be the regular ones.

**Review by U-DON — APPROVED** [review]

File: src/js/signin/components/Verify.jsx (hunk 2 above)
Comment (bshyong) [reply src/js/signin/components/Verify.jsx]: Yes will fix

**Pull request merged**